README.md

```diff
@@ -2,8 +2,8 @@
 
 ## Requirements
 
-* Xcode 13 or above
-* [Android and iOS environment setup](https://reactnative.dev/docs/environment-setup) described in the RN docs
+- Xcode 13 or above
+- [Android and iOS environment setup](https://reactnative.dev/docs/environment-setup) described in the RN docs
```

Comment:
`*` and `-` are [both supported](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists) for lists, so this change isn't necessary.